File: lessons/202/cs-app/Image.cs
internal readonly struct Image
{
    public string ObjKey { get; }
    public Guid ImageUuid { get; }
    public DateTime CreatedAt { get; }

    public Image()
    {
        ImageUuid = Guid.NewGuid();
        CreatedAt = DateTime.Now;
    }

    public Image(string key) : this()
    {
        ObjKey = key;
    }
}

File: lessons/201-quarkus-java-vs-fiber-go-performance-benchmark-in-kubernetes-201/java-app/src/main/java/com/antonputra/Image.java
package com.antonputra;

import java.util.UUID;
import java.time.LocalDateTime;

public class Image {

    public UUID imageId;
    public LocalDateTime createdAt;
    public String objKey;

    public Image(String objKey) {
        this.imageId = UUID.randomUUID();
        this.createdAt = LocalDateTime.now();
        this.objKey = objKey;
    }
}
